Sources/SourceKitBazelBSP/RequestHandlers/BuildTargets/BazelTargetQuerier.swift

```diff
@@ -77,12 +76,11 @@ final class BazelTargetQuerier {
         let providedTargetsQuerySet = "set(\(providedTargets.joined(separator: " ")))"
 
         // NOTE: important to sort for determinism
-        let topLevelKindsFilter = topLevelRuleKinds.sorted().joined(separator: "|")
         let dependencyKindsFilter = dependencyKinds.sorted().joined(separator: "|")
 
         // Collect the top-level targets -> collect these targets' dependencies
         let topLevelTargetsQuery = """
-            let topLevelTargets = kind("\(topLevelKindsFilter)", \(providedTargetsQuerySet)) in \
+            let topLevelTargets = kind("rule", \(providedTargetsQuerySet)) in \
```

Comment:
Basically the previous logic worked, but it meant we had to check if something was missing from the output to fully detect if the user messed up the inputs. I tried `rule` and we get the same output, but without filtering the wrong types (making it easier to detect them below)